### ISSUE TITLE
General: have OpenCOR signed

### ISSUE THREAD

**Issue by agarny** [opencor/opencor#1781]
Right now, if you download OpenCOR and run it on, say, Windows, then you will get something like:

![image](https://user-images.githubusercontent.com/602265/44570056-c81e1c80-a77c-11e8-9983-02f6b61e5aba.png)

So, you need to click on `Run anyway` to actually run OpenCOR. This only occurs the first time you run a downloaded version of OpenCOR, so it's not too bad. Still, we wouldn't get this warning if OpenCOR was signed. (The same applies on macOS, not sure on Linux.)

When it comes to signing an application, some useful information can be found for:
 - **Windows signing:** [here](https://blog.pakkly.com/how-to-sign-qt-apps-on-windows-for-release/) and [here](https://github.com/mintty/wsltty/issues/32#issuecomment-340450073).
 - **macOS signing:** [here](https://blog.pakkly.com/how-to-sign-qt-apps-on-mac-for-release/).

Things to do:
 - [x] **Windows:** [obtain an EV certificate](https://docs.microsoft.com/en-us/windows-hardware/drivers/dashboard/get-a-code-signing-certificate).
 - [x] **Apple:** [enroll to the Apple Developer Program](https://developer.apple.com/programs/enroll/).
 - Sign OpenCOR on:
   - [x] Windows; and on
   - ~macOS~.

**Comment by agarny** [comment]
- To sign on Windows is very easy since all that is needed is to sign the OpenCOR executable.
- It's a very different story when it comes to macOS since everything in the OpenCOR bundle must be signed, which is impossible (?) to do because of the >20k files to sign.
- I thought we could get away with it by moving our `Python` directory out of the OpenCOR bundle, but... then macOS complains about OpenCOR using our Team ID and our Python binaries using a different Team ID (well, no Team ID since they are not signed).
- The latter can be "fixed", but then there will always be the issue of a user installing "new" Python packages. Those won't be signed and will therefore cause problems.

**Bottom line:** we can sign OpenCOR on Windows, but not on macOS because it comes with its own copy of Python.

**Comment by agarny** [comment]
Reopening since... I wasn't signing in tne end!
